models/product.js

```diff
@@ -1,32 +1,33 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 
 const Schema = mongoose.Schema;
 
-const productSchema = new Schema({
+const productSchema = new Schema(
+  {
     name: {
-        type: String,
-        required: true,
-        unique: true,
+      type: String,
+      required: true,
     },
     category: {
-        type: String,
-        required: true,
+      type: String,
+      required: true,
     },
     color: {
-        type: String,
-        required: true,
+      type: String,
+      required: true,
     },
     price: {
-        type: Number,
-        required: true,
-    }, 
+      type: Number,
+      required: true,
+    },
     seller: {
-        type: Schema.Types.ObjectId,
-        ref: 'User',
-        required: true,
```

Comment:
We still need required here. it just can't be unique. otherwise this looks good.